packages/app-mobile/components/screens/NoteTagsDialog.js

```diff
@@ -140,16 +145,16 @@ class NoteTagsDialogComponent extends React.Component {
 				fontSize: 20,
 				color: theme.color,
 			},
-			newTagBox: {
+			tagBox: {
 				flexDirection: 'row',
 				alignItems: 'center',
-				paddingLeft: theme.marginLeft,
-				paddingRight: theme.marginRight,
+				paddingLeft: 10,
```

Comment:
Trying to make the left edge of the label aligns with the tags's left edge so it's more visually pleasing. But I'm fine with leaving this out.